drf_braces/serializers/enforce_validation_serializer.py

```diff
@@ -31,13 +31,25 @@ def run_validation(self, data=empty):
             if must_validate_fields is None or field_name in must_validate_fields:
                 raise
             else:
+                self.capture_failed_field(field_name, data)
```

Comment:
lets pass the exception as well which can provide exact error message what was wrong with the field